switch_model/wecc/get_inputs/register_post_process.py

```diff
@@ -8,17 +8,15 @@
 
 These 2 functions are kept in a separate file to avoid cyclical dependencies.
 """
+
+from functools import wraps
 import functools
 
 _registered_steps = {}
 
 
 def register_post_process(
-        name,
-        msg=None,
-        enabled=True,
-        only_with_config=False,
-        priority=2
```

Comment:
I'm guessing the intent is for the order to be determined by the order specified in the config file? If so, that makes sense to me.